autogen.sh

```diff
@@ -13,7 +13,6 @@ TEST_TYPE=-f
 FILE=lqr/lqr_carver.c
 
 AUTOCONF_REQUIRED_VERSION=2.54
-AUTOMAKE_REQUIRED_VERSION=1.6
```

Comment:
This line is removed but the variable `AUTOMAKE_REQUIRED_VERSION` is still used below.